new/PRS12948MalakaMe.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12948MalakaMe" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malʿaka Mǝḥrat Yāred Gǝrmāye</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-04-06">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">መልዓከ፡ ምሕረት፡ ያሬድ፡ ግርማዬ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Malʿaka Mǝḥrat Yāred Gǝrmāye</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">deacon</occupation>
+                    <occupation type="ecclesiastic">Administrator of Č̣alaqot Śǝllāse</occupation>
+                    <floruit when="1975"/>
+                    <residence>
+                        <placeName ref="LOC2091Calaqo">Č̣alaqot Śǝllāse</placeName>
+                    </residence>
+                </person>
+                <!--- <listRelation>
+                  Malʿaka Mǝḥrat Yāred Gǝrmāye owned the MS from which ms BDLaethd18 was copied: 
```

Comment:
I think we will need to talk about the use and way we approach the encoding of these documents at the meeting.